src/scripts/fields/info.coffee

```diff
@@ -36,6 +36,7 @@ Formbuilder.registerField 'info',
     $('.fb-info-editor').summernote(
         onChange: -> update.call(@)
         onKeyup: -> update.call(@)
+        disableDragAndDrop: true
```

Comment:
Any reason why we need disable drag and drop?

Reply:
Thanks Ray - this is actually a carry-over from ticket https://integralcs.atlassian.net/browse/INTEGRALCS-12091 .
When I did 12091, I didn't realise the formbuilder file was generated in another project. I added this line to the integral formbuilder.js file directly.
So whilee I was here, I needed to make sure my changes were updated in this repo to match it. Sorry I missed this the first time. I tested and confirmed that after compiling with grunt that the change to the formbuilder.js file is exactly the same as what I added to integral.